Display scaffolds in one of the six planes

ddjnw1yu opened the pull request:
Because the x/y/z axis directions are not always consistent across different scaffolds. Using `[±1, 0, 0]/[0, ±1, 0]/[0, 0, ±1]` may get a unexpected rotation. (however `[±1, 0, 0]/[0, ±1, 0]/[0, 0, ±1]` are still allowed to be used to adjust the Camera.)

The actual (viewport-related) plane axis will be calculated when `front/back/right...` is passed as a param. The displayed plane after the scaffold is loaded will be the front plane.

alan-wu committed:
Add a test alginCameraWithAxis method.

ddjnw1yu committed:
Merge remote-tracking branch 'upstream/axis' into axis

ddjnw1yu committed:
Allow both Vector 3 and Array as param

ddjnw1yu committed:
Return loaded scaffold plane axis

ddjnw1yu commented:
Thinking of combining these two functions. To allow string input options (front/back/left, etc.) instead of using a separate function to calculate the plane axis.

ddjnw1yu committed:
Combine two functions

alan-wu commented:
@ddjnw1yu I think you should separate them into three different functions instead, one for each function and one with them combined. I can see use case for all three functions.

ddjnw1yu committed:
Split function for different use cases

ddjnw1yu committed:
Rename variable

The vector3 for different direction should no be changed when viewport is updated. Rename to make it clearer.

ddjnw1yu commented:
No longer required.
